hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/HddsConfigKeys.java

```diff
@@ -289,7 +289,18 @@ private HddsConfigKeys() {
       "hdds.datanode.http-address";
   public static final String HDDS_DATANODE_HTTPS_ADDRESS_KEY =
       "hdds.datanode.https-address";
-
+  public static final String HDDS_DATANODE_CLIENT_ADDRESS_KEY =
+      "hdds.datanode.client.address";
+  public static final String HDDS_DATANODE_CLIENT_BIND_HOST_KEY =
+      "hdds.datanode.client.bind.host";
+  public static final String HDDS_DATANODE_CLIENT_BIND_HOST_DEFAULT =
+      "0.0.0.0";
+  public static final String HDDS_DATANODE_CLIENT_PORT_KEY =
+      "hdds.datanode.client.port";
+  public static final int HDDS_DATANODE_CLIENT_PORT_DEFAULT = 9864;
+  public static final String HDDS_DATANODE_HANDLER_COUNT_KEY =
+      "hdds.datanode.handler.count";
+  public static final int HDDS_DATANODE_HANDLER_COUNT_DEFAULT = 1;
```

Comment:
New config properties need to be added in `ozone-default.xml`, too.

```
... 4 variables missing in ozone-default.xml Entries:   hdds.datanode.client.address  hdds.datanode.client.bind.host  hdds.datanode.client.port  hdds.datanode.handler.count
```

https://github.com/xichen01/ozone/actions/runs/5075939913/jobs/9117643161#step:5:3864

Reply:
done.